providers/pinpoint/package.json

```diff
@@ -0,0 +1,88 @@
+{
+  "ava": {
+    "failFast": true,
+    "files": [
+      "!build/module/**"
+    ],
+    "timeout": "60s",
+    "typescript": {
+      "rewritePaths": {
+        "src/": "build/main/"
+      }
+    }
+  },
+  "dependencies": {
+    "@aws-sdk/client-pinpoint-sms-voice-v2": "3.231.0",
+    "@novu/stateless": "^0.9.0"
```

Comment:
We are currently in v.0.13.0 and this versioning and this provider version should be updated accordingly.

Reply:
Alright, I updated it.